specs/provider.yaml

```diff
@@ -1043,3 +1080,33 @@ definitions:
     additionalProperties: false
     required:
     - message
+
+  ResourceMeasures:
+    type: object
+    description: |
+      Wrapper object for resource measures body.
```

Comment:
We don't need to worry about wrapper objects for the provider API, it's not a standard we have for them, only one we have internally:
https://docs.manifold.co/providers#tag/Credential
No need to get them to do the extra work.

Reply:
I think it's important to remove this, but let me know if you think otherwise.

Reply:
Makes sense.